stories/chats/src/BubbleActionButton.tsx

```diff
@@ -41,13 +51,26 @@ export function BubbleActionButton(message: ActionButtonMessage) {
         return UIMsgButtonVariant.Neutral;
     }, [variant]);
 
+    const iconPosition = React.useMemo<UIMsgButtonIconPosition | undefined>(() => {
+        switch (iconPositionProp) {
+            case ActionButtonMessageIconPosition.Left:
+                return UIMsgButtonIconPosition.Left;
+            case ActionButtonMessageIconPosition.Right:
+                return UIMsgButtonIconPosition.Middle;
+            default:
+                return UIMsgButtonIconPosition.Left;
+        }
```

Comment:
Smth is wrong here, no?
If it's right, why should it be middle?

Reply:
Middle - it is right position from title and up from caption. Because there is no caption so it's ok.